packages/replay-internal/test/integration/errorSampleRate.test.ts

```diff
@@ -4,6 +4,7 @@
 
 import '../utils/mock-internal-setTimeout';
 import { captureException, getClient } from '@sentry/core';
+import type { MockInstance } from 'vitest';
```

Comment:
e.g. this was not imported but still worked, for whatever reason? Now it "correctly" complained about this.